homeassistant/components/meteo_france/weather.py

```diff
@@ -55,11 +66,12 @@ def __init__(self, coordinator: DataUpdateCoordinator, mode: str):
         self.coordinator = coordinator
         self._city_name = self.coordinator.data.position["name"]
         self._mode = mode
+        self._unique_id = f"{self.coordinator.data.position['lat']}/{self.coordinator.data.position['lon']}-weather"
```

Comment:
```suggestion
        self._unique_id = f"{self.coordinator.data.position['lat']},{self.coordinator.data.position['lon']}"
```

The unique_id is automatically prefixed by the platform, so we should not add it to unique_id.

Here the unique_id will be `weather.4.58956,73.1856`

Reply:
Are you sure for the `unique_id` ? I have followed the recommandation in the [developer documentation](https://developers.home-assistant.io/docs/entity_registry_index/#unique-id-requirements).

> If a device has a single serial but provides multiple entities, combine the serial with unique identifiers for the entities. For example, if a device measures both temperature and humidity, you can uniquely identify the entities using {serial}-{sensor_type}.

Reply:
Extreamly sure, I got a comment on a PR about that.

Reply:
about the doc, it's in the case you have multiple type of entities in a same platform

Reply:
done